### PR TITLE
Add text fadeIn and Make just cover clickable

### LINKED ISSUE #1712: Video preview improvements
- Make only the cover trigger the hover effect. Other than the cover, only the title should redirect to the video
- Make the appear transition (opacity) affect the text as well, not only the cover image

### PR DESCRIPTION
**Pull request by fcescob**
Fixes #1712

### CODE REVIEW

File: atlas/src/shared/components/VideoPreview/VideoPreview.styles.tsx [hunk 1]
```diff
@@ -91,22 +92,36 @@ export const StyledAvatar = styled(Avatar)<ChannelProps>`
   cursor: ${({ channelClickable }) => (channelClickable ? 'pointer' : 'auto')};
 `
 
-export const TitleHeader = styled.h3<MainProps & ScalesWithCoverProps>`
+const fadeIn = keyframes`
```
Comment (kdembler) [inline atlas/src/shared/components/VideoPreview/VideoPreview.styles.tsx]: Couldn't we add that animation to the `InfoContainer` in the `VideoPreviewBase.style.tsx`? Then we can fade in the whole container, not the individual nodes

File: atlas/src/shared/components/VideoPreview/VideoPreviewBase.styles.tsx [hunk 2]
```diff
@@ -31,16 +31,33 @@ export const CoverWrapper = styled.div`
...
   padding-top: 56.25%;
-
   transition-property: box-shadow, transform;
   transition-duration: 0.4s;
   transition-timing-function: cubic-bezier(0.165, 0.84, 0.44, 1);
   animation: ${fadeIn} 0.5s ease-in;
+  cursor: ${(props) => (props.clickable ? 'pointer' : 'auto')};
+  :hover {
+    ${(props) =>
+      props.clickable
```
Comment (kdembler) [inline atlas/src/shared/components/VideoPreview/VideoPreviewBase.styles.tsx]: I found it a good practice to separate conditional styles to a separate `css` constant. That way it's much easier to read the styles. I know this is copied but perhaps we could do that anyway?

File: atlas/src/shared/components/VideoPreview/VideoPreview.styles.tsx (hunk 1 above)
Comment (fcescob) [reply atlas/src/shared/components/VideoPreview/VideoPreview.styles.tsx]: That would be better indeed, but doing so changes the color of the placeholders inside `InfoContainer` when loading, so I suggest we keep this as is for now

File: atlas/src/shared/components/VideoPreview/VideoPreviewBase.styles.tsx (hunk 2 above)
Comment (fcescob) [reply atlas/src/shared/components/VideoPreview/VideoPreviewBase.styles.tsx]: Good point, will change that

File: atlas/src/shared/components/VideoPreview/VideoPreview.styles.tsx (hunk 1 above)
Comment (kdembler) [reply atlas/src/shared/components/VideoPreview/VideoPreview.styles.tsx]: Okay, we can skip that, although there is another problem. Currently the fade in animations aren't in sync, i.e. this uses different duration and easing than the cover. I suggest we define the keyframes and the animation in the base styles file and import it here so these parameters are always the same. Like so:

```js
const fadeIn = keyfreames...

export const fadeInAnimation = css`
  animation: ${fadeIn} 0.5s ease-in;
`
```

**Review by kdembler — APPROVED** [review]
LGTM

**Pull request merged**